packages/react-server/core/renderMiddleware.js

```diff
@@ -915,15 +915,16 @@ function bootstrapClient(res, lastElementSent) {
 	// function to avoid letting responses slip in between.
 	setupLateArrivals(res);
 
-	for (var i = 0; i <= lastElementSent; i++) {
-		wakeElement(res, i);
-	}
+	wakeElement(res, 0, lastElementSent);
```

Comment:
`wakeElement` seems a little mis-named now that it may wake _multiple_ elements. 😕